server/app/routes/api/v1/va/serializers.js

```diff
@@ -104,8 +104,8 @@ function serializeAmbassadorForAdmin(ambassador) {
   obj["onboarding_completed"] = ambassador.get("onboarding_completed")
   obj["giftcard_completed"] = ambassador.get("giftcard_completed")
   obj["signup_completed"] = ambassador.get("signup_completed")
-  obj["locked"] = isLocked(ambassador);
   obj["payout_provider"] = ambassador.get("payout_provider")
+  obj["admin_bonus"] = ambassador.get("admin_bonus") ? ambassador.get("admin_bonus").toNumber() : ""
```

Comment:
is toNumber still appropriate now that admin_bonus is being represented as a number?

Reply:
updated